ui/k-info-panel/show_telemetry_int_data.kytos

```diff
@@ -0,0 +1,522 @@
+                request.done(function(data) {
+                });
+                request.fail(function(data) {
+                    let notification = {
+                        icon: 'cog',
+                        title: 'Could not disable INT on EVCs',
+                        description: data.status.toString() + ": " + data.responseJSON.description
+                    }
+                    self.$kytos.eventBus.$emit("setNotification", notification)
+                });
+            },
+            //API request to redeploy EVCs selected within UI
+            redeploySelectedEVCs() {
+                var self = this
+                let evc_id = []
+                for (const EVC in this.content) {
+                    evc_id.push(EVC.toString())
+                }
+                let payload = {evc_ids: this.selectedEVCs}
+                let request = $.ajax({
+                                type:"PATCH",
+                                dataType: "json",
+                                contentType: "application/json",
+                                data: JSON.stringify(payload),
+                                url: this.$kytos_server_api + "kytos/telemetry_int/v1/evc/redeploy",
+                                async: true});
+                request.done(function(data) {
+                });
+                request.fail(function(data) {
+                    let notification = {
+                        icon: 'cog',
+                        title: 'Could not redeploy INT on EVCs',
+                        description: data.status.toString() + ": " + data.responseJSON.description
+                    }
+                    self.$kytos.eventBus.$emit("setNotification", notification)
+                });
+            }
+        },
+        computed: {
+            //Filters the content obtained via the prop from main.kytos based on the textFilters/searchColumns and then the column ordering
+            filtered_EVCData: function() {
+                let current_data = this.$kytos.toRaw(this.content)
+                //Filter based on the search columns
+                if (current_data) {
+                    let filtered_data = current_data.filter((EVC) => {
+                        for (const tableColumn of this.EVCTextFilter) {
+                            let separatedFilter = tableColumn[1].toString()
+                            let separatedProperty = EVC[tableColumn[0]].toString()
+                            if (!(separatedProperty.includes(separatedFilter) || separatedFilter.includes(separatedProperty))) {
+                                return false
+                            }
+                        }
+                        return true
+                    })
+                    current_data = filtered_data
+                }
+                //Sort by ascending or descending order
+                switch(this.sortedColumnName[0]) {
+                    case "ID":
```

Comment:
Similar to the prior comment, I'll recommend that you try to replace these switch cases conditionals with a new method which will take `name` and the `sort direction` (`this.sortedColumnName[1]`), and then you can use a JS object or a hashmap mapping the `name` to which attribute it should compare like: `{ID: "id", NAME: "name"}` (or even easier since they follow a pattern just lower the string, an then in the `sort` closure you lookup for that attribute dynamically `a[attr]` < `b[attr]` or `a[attr]` > `b[attr]`. 

See what I mean? The current code works, but let's try to make it a bit more concinse.